Core/src/org/sleuthkit/autopsy/commonpropertiessearch/CommonAttributeSearchAction.java

```diff
@@ -151,7 +151,7 @@ protected void done() {
     }
 
     @NbBundle.Messages({
-        "CommonAttributeSearchAction.getName.text=Find Common Properties"})
+        "CommonAttributeSearchAction.getName.text=Find Related Cases"})
```

Comment:
I think we want the dialog over at line 94 of CommonAttributePanel.java to have the same name as the action which opens it.